modules/swiss/src/main/model.scala

```diff
@@ -28,3 +28,42 @@ case class FeaturedSwisses(
 )
 
 case class SwissFinish(id: Swiss.Id, ranking: Ranking)
+
+object SwissBounds {
+  val maxRounds           = 100
+  val maxScore            = maxRounds * 2
+  val maxBuccholz         = maxScore * 2
```

Comment:
should this not be maxScore * maxRounds for both buccholz and sonnenbornberger?
as you can get 100 scores of 200 as an upper bound (really 100 scores of 198).

Reply:
Yes!  Lol

Reply:
Pushed. :D